content/blog/gen-ai-for-pm-series-part-4-multi-agent-systems-for-multi-stakeholder-projects.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Multi-agent systems for multi-stakeholder projects
+date: 2025-08-30T06:50:31.006Z
+author: Dinesh R Singh, Nisha Rajput, Varsha Shekhawat
+authorimage: /img/blog-image.jpeg
+disable: false
+tags:
+  - Gen AI
+  - AI agent
+  - Agentic AI
+  - AI project manager
+  - Scrum master
+  - Multi-agent
+  - "Multi-stakeholder "
+---
+<style>
+
+li {
+
+   font-size: 27px;
+
+   line-height: 33px;
+
+   max-width: none;
+
+}
+
+</style>
+
+*Part 4 of our **Gen AI for PM series***
+
+## Intro
+
+Large projects often involve multiple stakeholders, each with different priorities, risks, and information needs. Managing all of these moving parts with a single tool or process quickly becomes overwhelming. This is where **multi-agent systems** come in.
+
+By assigning specialized AI agents to handle distinct responsibilities such as scheduling, compliance, risk management, or communication projects can scale more smoothly. Each agent works autonomously within its domain but shares insights with others, creating a **coordinated ecosystem** that adapts to stakeholder demands in real time. For project managers, this means fewer blind spots, faster responses, and better alignment across diverse teams.
+
+**Coordinating across departments is chaotic**
+
+If you’ve ever managed a cross-department project, you know how messy things can get. Finance is tracking budgets in spreadsheets. Engineering is sprinting toward deadlines in Jira. Marketing is waiting for updates before they can launch campaigns. And operations? They’re buried in vendor calls.
+
+What ties it all together? Usually, it’s the project manager stuck in the middle — drowning in emails, calendar invites, and status requests.
+
+This **chaos of coordination** is where **multi-agent systems** step in. Instead of a single PM being the communication bottleneck, digital agents act like specialized assistants, each handling a slice of responsibility and keeping everyone aligned.
+
+A cartoon-style illustration of a PM juggling multiple balls labeled Finance, Engineering, Marketing, Operations — beside them, AI agents smoothly passing the same balls around in sync.
```

Comment:
Remove this statement. Instead, you might consider changing the titles of the images from "PM" and "AI AGENTS" to "Without AI agents" under the left one and "With AI agents" or "With Agentic AI" under the right one.

The statement that I recommended you remove would actually work quite well as an "Alt text" for the image, should you decide to provide that and make this file more accessible for people who experience challenges with vision and rely upon e-readers. But that would not be placed here... that would be a part of the images meta data.